### PR TITLE
Feat: Styles of `Error.tsx` shifted in tailwindcss

### LINKED ISSUE #535: Feat: converting `error.css` file into tailwind classes
**Is your feature request related to a problem? Please describe.**
Even website codebase is shifted to using tailwind CSS utility classes the  files `utils/Error.tsx` still uses the vanilla CSS imported from `error.css` file. 

**Describe the solution you'd like**
To transfer Vanilla CSS  to tailwind CSS utility classes in Error.tsx file.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by DhairyaMajmudar**
**Task Done**

- Rewritten vanilla CSS styles into tailwind utility classes.
- Converted HTML button into ShandCn button
- deleted `error.css` file.
- corrected import path of Error.tsx file in `PublicRoute.tsx` file.

closes #535 
related to #536 

**Screenshot**

image

### CODE REVIEW

File: website/src/pages/PublicRoute.tsx [hunk 1]
```diff
@@ -43,7 +43,10 @@ const PublicRoute = () => {
         <Route path="/search" element={<SearchPage />} />
         <Route path="/compare" element={<ComparePage />} />
         <Route path="/macro" element={<MacroPage />} />
-        <Route path="/macrobench/queries/compare" element={<MacroQueriesComparePage />} />
+        <Route
+          path="/macrobench/queries/compare"
+          element={<MacroQueriesComparePage />}
+        />
```
Comment (frouioui) [inline website/src/pages/PublicRoute.tsx:49]: Any reason for this change?

File: website/src/utils/Error.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2023 The Vitess Authors.
```
Comment (frouioui) [inline website/src/utils/Error.tsx:2]:
```suggestion
Copyright 2024 The Vitess Authors.
```

**Review by frouioui — APPROVED** [review]
Looks good to me but left a question.

File: website/src/pages/PublicRoute.tsx (hunk 1 above)
Comment (DhairyaMajmudar) [reply website/src/pages/PublicRoute.tsx:49]: Seems like this change is done by prettier extension of vscode. This is not required.

Comment (DhairyaMajmudar) [reply website/src/pages/PublicRoute.tsx]: I have reverted back that change 👍🏻

**Review by frouioui — APPROVED** [review]
LGTM, thank you!

**Pull request merged**